src/Popover/Popover.scss

```
@@ -1,2 +1,50 @@
 @import "variables";
 @import "~bootstrap/scss/popover";
+
+.popover {
+    filter: $popover-box-shadow;
+
+    .popover-header {
+        display: flex;
+    }
+
+    .pgn__icon {
+        margin-right: $popover-icon-margin-right;
```

The icon in the design is forced to be 16px by 16x.

```suggestion
        margin-right: $popover-icon-margin-right;
        height: 1rem;
        width: 1rem;
```